tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1741,3 +1756,48 @@ def test_positive_group_sync_open_ldap_authsource(
         assert session.activationkey.search(ak_name)[0]['Name'] == ak_name
         current_user = session.activationkey.read(ak_name, 'current_user')['current_user']
         assert user_name.capitalize() in current_user
+
+
+@tier2
+def test_verify_group_permissions(
+    session, auth_source_ipa, multigroup_setting_cleanup, groups_teardown, ldap_tear_down
+):
+    """Verify group permission
+
```

Comment:
Add some more details in the docstring, `Verify group permission` does not set the context

Reply:
Done